ClientApp/src/app/components/result-home/result-home.component.html

```diff
@@ -3,15 +3,14 @@
       <div id="head-page-section" class="l-grey-bg head-page-section">
        <div class="container">
          <div class="row">
-           <div class="col-md-5 col-sm-7">
+           <div class="col-md-4 col-sm-4 col-xs-12">
              <div class="head-title">
                <h1>
                  Resultados
                </h1>
-
              </div> <!-- end subsection-text -->
            </div> <!-- end col-md-8 -->
-           <div class="col-md-7 offset-md-2 text-right">
+           <div class="col-md-8 col-sm-8 col-xs-12 text-right">
              <button type="button" class="btn btn-primary btn-md" (click)="openModal(generator)">
```

Comment:
This method doesn't exist.

Reply:
![screenshot_5](https://user-images.githubusercontent.com/19880333/43562008-f1e259ce-95e7-11e8-86c3-28573c2ef9b1.png)

Reply:
Okay, it was probably lost in a merge then. I can confirm the existence of that method in staging too.